texajson.go

```diff
@@ -213,6 +278,28 @@ func ConvtoSlabPage(ArtiQSA []uint64, SlabNameArray []string, slabSeqArray []str
 				}
 			}
 		}
+
+		for j := 0; j < len(SlabNameArray); j++ {
+			var intractn Interaction
```

Comment:
Can you come up with a shorter name like `intn` instead?

Reply:
Also, using `interaction` should just be fine.

Reply:
Done. using interaction now.